mocha-casperjs.js

```diff
@@ -48,7 +48,6 @@ module.exports = function (Mocha, casper, utils) {
   })
 
   casper.on('waitFor.timeout', function(timeout, details) {
-    casper.step++
```

Comment:
there are two more places with `casper.step++`. Are these commands required? Should it be deleted too?